tests/test_coordinators.py

```
@@ -3554,6 +3641,42 @@ async def test_client_action_uses_mapped_site_name(
+                    "id": "device1",
+                    "_id": "60a1b2c3d4e5f67890123456",
+                    "macAddress": "AA:BB:CC:DD:EE:FF",
+                }
+            }
+        }
+        facade_coordinator._device_coordinator._legacy_site_names = {"site1": "branch"}
+        facade_coordinator.network_client.devices.set_outlet_state = AsyncMock(
+            return_value=True
+        )
+
+        result = await facade_coordinator.async_set_outlet_state(
```

Pylint (4.0.7)

[warning] Unused variable 'result'

(W0612)